diff --git a/node_modules/zego-express-engine-reactnative/lib/impl/ZegoExpressEngineImpl.js b/node_modules/zego-express-engine-reactnative/lib/impl/ZegoExpressEngineImpl.js
index b86e4e1..bed1f96 100644
--- a/node_modules/zego-express-engine-reactnative/lib/impl/ZegoExpressEngineImpl.js
+++ b/node_modules/zego-express-engine-reactnative/lib/impl/ZegoExpressEngineImpl.js
@@ -2,7 +2,7 @@
 import { NativeModules, NativeEventEmitter } from 'react-native';
 import * as zego from '../ZegoExpressDefines';
 const { ZegoExpressNativeModule } = NativeModules;
-const Prefix = ZegoExpressNativeModule.prefix;
+const Prefix = ZegoExpressNativeModule ? ZegoExpressNativeModule.prefix : '';
 const ZegoEvent = new NativeEventEmitter(ZegoExpressNativeModule);
 let engine;
 export class ZegoExpressEngineImpl {
